services/ui-src/src/routes/ReadOnly/ReadOnly.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import userEvent from "@testing-library/user-event";
+// components
+import { ReadOnly } from "routes";
+// utils
+import { mockAdminUser, RouterWrappedComponent } from "utils/testing/setupJest";
+import { useUser } from "utils";
+
+// MOCKS
+
+jest.mock("utils/auth/useUser");
+const mockedUseUser = useUser as jest.MockedFunction<typeof useUser>;
+
+const mockVerbiage = {
+  header: "mock header",
+  buttonLabel: "mock button label",
+};
+
+const readOnlyView = (
+  <RouterWrappedComponent>
+    <ReadOnly verbiage={mockVerbiage} />
+  </RouterWrappedComponent>
+);
+
+const fillOutForm = async (form: any) => {
+  // selectors for all the required fields
+  const dropdownInput = form.querySelector("[name='ads-state']")!;
+  // fill out form fields
+  fireEvent.change(dropdownInput, { target: { value: "CA" } });
```

Comment:
Should add an await here.
```suggestion
  await fireEvent.change(dropdownInput, { target: { value: "CA" } });
```